index.html

```diff
@@ -13,10 +13,10 @@
     </head>
     <body>
         <header class="navbar navbar-expand sticky-top navbar-dark bg-info flex-column flex-md-row" id="header">
+            <img class="img-right mr-2" src="GWF_logo.png" height="30"/>
```

Comment:
The logos seem pretty small to me now. The "GWF" is hardly readable on my screen. 36 would be readable.

Reply:
Addressed.